SPECS/python-ssh2-python/python-ssh2-python.spec

```diff
@@ -0,0 +1,138 @@
+%{?python_enable_dependency_generator}
+%global debug_package %{nil}
+
+%global modname ssh2-python
+
+Name:           python-%{modname}
+Version:        0.27.0
+Release:        1%{?dist}
+Summary:        Super fast SSH library - bindings for libssh2
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/ParallelSSH/ssh2-python
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/ssh2-python-%{version}.tar.gz
+Source1:        conftest.py
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  libssh2-devel
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-Cython
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+
+%if %{with_check}
+BuildRequires:  python3-pytest
+BuildRequires:  %{_sbindir}/sshd %{_bindir}/ssh-agent
```

Comment:
Please put these build requirements on separate lines for readability

Reply:
Done